src/main/java/frc/robot/RobotContainer.java

```diff
@@ -146,6 +149,23 @@ private void configureButtonBindings() {
                             new Pose2d(drive.getPose().getTranslation(), new Rotation2d())),
                     drive)
                 .ignoringDisable(true));
+
+    dec.setDefaultCommand(dec.runOnPID());
+    Trigger x_bot = controller2.cross();
+    x_bot.onTrue(Commands.runOnce(() -> dec.setTarget(0, 0)));
+    Trigger circle_bot = controller2.circle();
```

Comment:
after cleanup on ICEE and nDexter, need to revisit which commands are actually needed